docs/rules/no-mixed-operators.md

```diff
@@ -0,0 +1,125 @@
+# Disallow mixes of different operators (no-mixed-operators)
+
+Enclosing complex expressions by parentheses makes developer's intention clarifying, then the code would get more readability.
+This rule warns that there are different operators consecutively without parentheses *in AST*.
+
+```js
+var foo = a && b || c || d;    /*BAD: Unexpected a mix of '&&' and '||'.*/
+var foo = (a && b) || c || d;  /*GOOD*/
+var foo = a && (b || c || d);  /*GOOD*/
+```
+
+## Rule Details
+
+This rule checks `BinaryExpression` and `LogicalExpression`.
+
+This rule would conflict with [no-extra-parens] rule.
+If you use both this and [no-extra-parens] rule together, you need to use `nestedBinaryExpressions` option of [no-extra-parens] rule.
```

Comment:
This rule MAY conflict with [no-extra-parens] rule.
If you use both this and [no-extra-parens] rule together, you need to use THE `nestedBinaryExpressions` option of [no-extra-parens] rule.